code/modules/mob/living/carbon/human/human.dm

```diff
@@ -288,8 +288,8 @@
 
 				while(limbs_affected != 0)
 					processing_dismember = pick(organs)
-					if(processing_dismember.name != "chest" && processing_dismember.name != "groin" && processing_dismember.name != "head")
-						processing_dismember.droplimb(1,1,1)
+					if(processing_dismember.name != "chest" && processing_dismember.name != "head")
+						processing_dismember.droplimb(1)

```

Comment:
For the explosions, I'd make them all amputations technically; as part of an earlier explosion rework, explosion damage was upped quite dramatically to the mob itself and applied in a much more logical fashion; if it's a dropped limb it will transfer the max damage to the parent organ, which can make a light explosion's normal 30 damage something like 100+ if it happens to pick your legs to blow off.